### PR TITLE
load_balancing.html

### PR DESCRIPTION
**Pull request by bagnaram**
I added a section regarding the use of AWS ELB load balancer in a HA confguration. I refactored the lessons learned write-up and made the procedures more passive to address its use as a general user's guide.

### CODE REVIEW

**Comment by mdanter** [comment]
Most/all of this can be automated with terraform (open source tool written in go from hashicorp), and it’s also easy to create templates that only depend on aws id/key. Terraform also has providers for the major cloud providers, and openstack.

**Comment by etsauer** [comment]
adding @sabre1041 and @JaredBurck. I think we need to put this under another heading, maybe "Guides for Specific Load Balancer Implementations"

**Comment by bagnaram** [comment]
I fixed the content to be under its on heading as reccommended

File: playbooks/installation/load_balancing.adoc [hunk 1]
```diff
@@ -327,3 +327,62 @@ node02.myorg.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
 ===== 3. Run the Advanced Installer
 
 Run the installer per the instructions in the link:https://docs.openshift.com/enterprise/3.1/install_config/install/advanced_install.html#running-the-advanced-installation[Advanced Installation Guide]
+
+=== Guides for Specific Load Balancer Implementations
+
+===== 1. Load Balancing in Amazon Web Services
```
Comment (etsauer) [inline playbooks/installation/load_balancing.adoc]: Don't jump from level 3 to level 5/

**Comment by bagnaram** [comment]
I added procedures to configure the master HAProxy to log OpenShift Master API events.

**Comment by bagnaram** [comment]
bump

**Comment by etsauer** [comment]
@bagnaram sorry this fell through. I would like to get the merged. Can you resolve the conflicts in your branch so we can move forward?

**Comment by bagnaram** [comment]
Yes, will do. I have a couple updates to make for the latest OCP release. I will commit them later this afternoon. Thanks!

**Comment by bagnaram** [comment]
Changes are all set now

**Comment by JaredBurck** [comment]
thanks @bagnaram reviewing now

**Comment by JaredBurck** [comment]
@bagnaram @etsauer LGTM

**Pull request merged**